### ISSUE TITLE
Cooldown between squid grabs

### ISSUE THREAD

**Issue by ammoore00** [BTW-Community/BTW-Public#175]
Right now multiple squids can headcrab you back to back, making it almost impossible to escape. There should be a cooldown between killing one squid and when the next squid can attach itself to give the player a chance to gain their bearings, fight back, or attempt to escape.

**Comment by PlasmaFox256** [comment]
nah its all good you dont need to make the game easier cuz i bitched about getting combo-ed by squids
ive found a method to basically go through squid armies easily
just keep holding spacebar to float up and prevent yourself from drowning and just keep swining your weapon 2 times a second.
squids are easy to kill :)

**Comment by AbbyRead** [comment]
I disagree with PlasmaFox, as I think it is a valid criticism.  The fact that it confuses basically everyone at first and makes them think squids are nearly indestructible is unclear (and arguably unfair) game design.  Valid for fixing, even if the timer to put in is a short one.  👍

**Comment by Hirachosan** [comment]
Adding a squid death sound would go a long way as well here

Their constant chomping and being grabbed by a new one are very similar sounds if not the same. Having better auditory discern to know what's going on would help immensely seeing as all visual input is removed. 

Even with a delay, if the player is in the gloomy waters you most likely still have no clue what's going on. and some clues are definitely in order to get a grip on how you can improve and what's going on
